dataprofiler/profilers/column_profile_compilers.py

```diff
@@ -226,3 +231,23 @@ def profile(self):
             profile["data_label"] = profiler.data_label
             profile["statistics"].update(profiler.profile)
         return profile
+
+
+class UnstructuredCompiler(BaseCompiler):
+
+    # NOTE: these profilers are ordered. Test functionality if changed.
+    _profilers = [
+        TextProfiler,
+        UnstructuredDataLabelerProfile,
```

Comment:
I think maybe we should have uniformity in terms of Unstructured Profile class names and type (like the variable "type"). Maybe both labeled with the Unstructured prefix?

Reply:
Possibly, could we address this after this PR? I'll create an issue